firefox-ios/Client/Coordinators/Scene/SceneSetupHelper.swift

```diff
@@ -26,10 +26,16 @@ struct SceneSetupHelper {
     }
 
     func createNavigationController() -> UINavigationController {
-        let navigationController = UINavigationController()
+        let navigationController = RootNavigationController()
         navigationController.isNavigationBarHidden = true
         navigationController.edgesForExtendedLayout = UIRectEdge(rawValue: 0)
 
         return navigationController
     }
 }
+
+class RootNavigationController: UINavigationController {
```

Comment:
nit; Should this go in its own file?

Reply:
curious on what's the team opinion on creating new files? For me its easy if everything is in one area without having to jump around if it's small, but happy to always create a new file for anytime a class or struct if that is preferred.

Reply:
discussed on Slack, will leave as is here, but going forward will keep in mind to create new files for struct / classes.